R-package/DESCRIPTION

```diff
@@ -0,0 +1,28 @@
+Package: RGF
+Type: Package
+Title: Regularized Greedy Forest
+Version: 1.0.4
+Date: 2018-07-22
+Authors@R: c( person("Lampros", "Mouselimis", email = "mouselimislampros@gmail.com", role = c("aut", "cre")), person("Ryosuke", "Fukatani", role = "cph", comment = "Author of the python wrapper of the 'Regularized Greedy Forest' machine learning algorithm"), person("Tong", "Zhang", role = "cph", comment = "Author of the 'Regularized Greedy Forest' and of the Multi-core implementation of Regularized Greedy Forest machine learning algorithm"), person("Rie", "Johnson", role = "cph", comment = "Author of the 'Regularized Greedy Forest' machine learning algorithm") )
+Maintainer: Lampros Mouselimis <mouselimislampros@gmail.com>
+BugReports: https://github.com/mlampros/RGF/issues
```

Comment:
Please update the link with `https://github.com/RGF-team/rgf/issues`

Reply:
fixed and pushed changes